### PR TITLE
#187300189 Configure storybook for component development

### PR DESCRIPTION
**Pull request by janet-barbie**
#### What does this PR do?
StoryBook configuration
#### Description of Task to be completed?
Configure the storybook tool to make the working on related components more efficient, easier to reuse and have better component documentation.
#### What are the relevant pivotal tracker/Trello stories?
https://www.pivotaltracker.com/n/projects/2698211
#### Screenshots (if appropriate)
Screenshot 2024-05-28 at 15 39 21

### CODE REVIEW

**Review by k3lly003 — APPROVED** [review]

**Review by tchamianest — APPROVED** [review]

File: src/stories/button.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.storybook-button {
+  font-family: 'Nunito Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
```
Comment (Eli250) [inline src/stories/button.css:2]: Use tailwind for instead of pure css. The css files are not necessary

File: src/stories/header.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
+.storybook-header {
+  font-family: 'Nunito Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
```
Comment (Eli250) [inline src/stories/header.css:2]: Get rid of this file and use tailwing instead

File: src/stories/page.css [hunk 3]
```diff
@@ -0,0 +1,69 @@
+.storybook-page {
+  font-family: 'Nunito Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
```
Comment (Eli250) [inline src/stories/page.css:2]: Use tailwind instead

**Review by Eli250 — CHANGES_REQUESTED** [review]

File: lib/tailwind.css [hunk 4]
```diff
@@ -0,0 +1,9 @@
+.btn {
```
Comment (Eli250) [inline lib/tailwind.css:1]: We already have tailwind configuration file, why are you recreating it here?

File: lib/Button/Button.tsx [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import React, { ReactNode } from 'react'
```
Comment (Eli250) [inline lib/Button/Button.tsx:1]: Components should be under components directory unless it is a special pattern required from storybook.

**Review by Eli250 — CHANGES_REQUESTED** [review]

**Pull request merged**